pkg/codegen/util_go.go

```diff
@@ -0,0 +1,174 @@
+package codegen
+
+import (
+	"bytes"
+	"fmt"
+	"go/format"
+	"go/parser"
+	"go/token"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/dave/dst"
+	"github.com/dave/dst/decorator"
+	"github.com/dave/dst/dstutil"
+	"golang.org/x/tools/imports"
+)
+
+type GenGoFile struct {
+	Path   string
+	Walker dstutil.ApplyFunc
+	In     []byte
+}
+
+func PostprocessGoFile(cfg GenGoFile) ([]byte, error) {
```

Comment:
it's a problem for later, but - ugh i hate the duplication of this between here and thema

Reply:
I can open a PR in thema to expose it if you want :)

Reply:
See https://github.com/grafana/thema/pull/191